### ISSUE TITLE
Documentation gaps and the quick start

### ISSUE THREAD

**Issue by ns-rlewkowicz** [k8snetworkplumbingwg/sriov-network-operator#697]
@SchSeba This is an awesome product and easy to use. I think I made it way harder on myself than it needed to be. I might just be a goofy man. 

TLDR: The quick start should lead with helm, and I'd be happy to write something up if we want it. 

The quickstart starts with this make system. I've been doing this for 15 years now, recently been working with a ton of C++ so make has been my life. I'm no stranger to custom engineering and esoteric build systems. So I just took it at face value and started hacking away. 

Maybe it seems like common sense, theres pages of docs hundreds of lines of code. I might be a goof, I feel like it's easy to miss. There's a helm chart! And it works very nicely. 

I was wasting TONS of time hacking around the make system and building custom images. And it's all just right there... modified some values and it worked great

**Comment by SchSeba** [comment]
Hi @danielskowronski there is an open PR to change the documentation
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/925
if you have time please review and let me know on the PR if you have some comments.

**Comment by SchSeba** [comment]
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/925 is merged closing this one feel free to re-open it
